ftwhttp/request.go

```diff
@@ -103,8 +104,14 @@ func (r *Request) AddHeader(name string, value string) {
 // AddStandardHeaders adds standard headers to the request, if they don't exist
 //
 // This will add Content-Length and the proper Content-Type
-func (r *Request) AddStandardHeaders(size int) {
-	r.headers.AddStandard(size)
+func (r *Request) AddStandardHeaders() {
+	// For better performance, we always close the connection (unless otherwise)
+	r.headers.Add("Connection", "close")
```

Comment:
Set only if the `Connection` header isn't already present.

Reply:
Updated by https://github.com/coreruleset/go-ftw/pull/135/commits/84028c84b876ff6c60490572a97c5a946d12868c